Operativ/Controllers/PreviousMonthsController.cs

```diff
@@ -9,25 +9,32 @@ namespace Operativ.Controllers
     public class PreviousMonthsController : ApiController
     {
 
-        private IMongoCollection<Month> Collection { get; set; }
+        private IMongoCollection<Month> сollectionBisc { get; set; }
+        private IMongoCollection<Month> collectionIsc { get; set; }
 
         public PreviousMonthsController()
         {
             var client = new MongoClient();
-            var database = client.GetDatabase("Operation");           
-            Collection = database.GetCollection<Month>("previousMonth");
+            var database = client.GetDatabase("Operativ");
+            сollectionBisc = database.GetCollection<Month>("BISC");
+            collectionIsc = database.GetCollection<Month>("ISC");
         }
 
         public async Task Get(string id)
         {
-            for (int i = 3; i <= 14; i++)
+            var options = new UpdateOptions { IsUpsert = true };
+            int i = 0;
+            while (i != 2)
             {
-                
-                var dateInput = Parser.Parse(id, i);
-                var options = new UpdateOptions { IsUpsert = true };
-                var filter = Builders<Month>.Filter.Eq(m=>m.YearMonth, dateInput.YearMonth);
-                if (dateInput.Persent == "до грудня \r\n\t\tпопереднього року") break;
-                await Collection.ReplaceOneAsync(filter, dateInput, options);             //insert and update date in DB
+                for (int parseLineNumber = i + 2; parseLineNumber <= i + 13; parseLineNumber++)
+                {
+                    var dateInput = Parser.Parse(id, parseLineNumber, i);
+                    var filter = Builders<Month>.Filter.Eq(m => m.YearMonth, dateInput.YearMonth);
+                    if ((dateInput.Percent == "до грудня \r\n\t\tпопереднього року") || (dateInput.Percent == "&nbsp;")) break;
+                    if (i == 0) await collectionIsc.ReplaceOneAsync(filter, dateInput, options);
```

Comment:
Two similar lines of code. First define an affected collection. Then perform the operation.